package main

func main() {
	panic("runtime error")
}
